RFC: Demonstration of `http2-jetty`, `http3-jetty` & `http2-robaho`

mechite opened the pull request:
@SentryMan
Related-to: #46
Signed-off-by: Mahied Maruf <contact@mechite.com>

mechite committed:
RFC: Demonstration of `http2-jetty`, `http3-jetty` & `http2-robaho`

Signed-off-by: Mahied Maruf <contact@mechite.com>

SentryMan commented:
yeah the jetty spi is nonfunctional with http2/http3 atm

mechite committed:
Merge branch 'master' into master

SentryMan commented:
the robaho one is so simple I don't think we need a test, for jetty, what's needed is to make even a basic get call

mechite committed:
Merge branch 'master' into master

mechite committed:
Merge branch 'master' into master

mechite committed:
Add test (fails)

        Caused by: java.lang.ExceptionInInitializerError:
        Exception java.io.UncheckedIOException:
        java.io.IOException:
                jetty server is not bound to port 8080
                [in thread "main"]

Signed-off-by: Mahied Maruf <contact@mechite.com>

mechite commented:
@SentryMan I don't know if this stuff is stale by now for me to add tests months later, but yeah in 08d2284 I was getting weird issues I couldn't figure out, I set port 8080 and I think I get some kind of issue in `io.avaje.jex.core.BootstrapServer` or something, "jetty server is not bound to port <...>"

If you wanna, you (or I) could fork out the Robaho stuff from this branch and merge it if you want it

mechite commented:
closing for #430, the rest I guess is stale/dead for now
